packages/svelte/src/compiler/phases/3-transform/client/utils.js

```diff
@@ -311,3 +311,44 @@ export function create_derived_block_argument(node, context) {
 export function create_derived(state, arg) {
 	return b.call(state.analysis.runes ? '$.derived' : '$.derived_safe_equal', arg);
 }
+
+/**
+ * @param {import('./types.js').ComponentClientTransformState} state
+ * @param {string} quasi_to_add
+ */
+export function push_template_quasi(state, quasi_to_add) {
```

Comment:
why is this called quasi and is this now the only function that must be used to add to state.template?

If it is, maybe it could be added as a function to state.template itself so that at the usage site you just do `state.template.append(...)` or sth. that would read much cleaner

Reply:
yes, that's much nicer. I've updated it to do that